app/api/utils/specs/date.spec.js

```diff
@@ -25,4 +25,11 @@ describe('date helper', () => {
       );
     });
   });
+
+  describe('add years to current date', () => {
+    const today = new Date();
```

Comment:
I think hardcoded dates could be "more predictable" and prevent possible flakinesses? What do you think?

Reply:
I'm using this utility method specifically to add to a year to the current date, I'm not sure how to hardcoded it in tests.
I could change the function passing a fixed date and then add it 1 year fixed, but I don't know if it could be useful in other contexts.